addons/medical_status/functions/fnc_handleKilled.sqf

```diff
@@ -41,6 +41,14 @@ if (_causeOfDeath != "#scripted") then {
         _this set [1, _killer];
         _this set [2, _instigator];
     };
+} else { // Handle setDead being skipped (forced respawn)
+    
+    _unit setVariable [VAR_HEART_RATE, 0, true];
+    _unit setVariable [VAR_BLOOD_PRESS, [0, 0], true];
+
+    _unit setVariable [QEGVAR(medical,causeOfDeath), _reason, true];
```

Comment:
`_reason` undefined?
Why not to call `fnc_setDead` with reason `#scripted`?

Reply:
_reason being undefined is a typo, fixing

handleKilledMission runs whenever any entity dies, including remote entities
handleKilled only runs on a local unit's death

Both the EH's will run when a local unit dies, but handleKilled will only run when a local unit dies, so it's preferable that it handle things.